### DISCUSSION TITLE
Releases from 'github-actions'?

### DISCUSSION THREAD

**Discussion by onchov** [Eloston/ungoogled-chromium#2002] [General]
Hi all, first-timer in the discussions here with a question (apologies if obvious etc, concise replies are OK): why are the two latest releases of ungoogled-chromium for Windows (namely [102.0.5005.115-1](https://ungoogled-software.github.io/ungoogled-chromium-binaries/releases/windows/64bit/102.0.5005.115-1) and [103.0.5060.53-1](https://ungoogled-software.github.io/ungoogled-chromium-binaries/releases/windows/64bit/103.0.5060.53-1)) coming from the author [github-actions](https://github.com/github-actions)? It raises a flag for me because I can't '[view all releases from user](https://github.com/github-actions/ungoogled-chromium-binaries/releases)' and gauge the credibility of the source. Cheers.

**Comment by networkException** [comment]
The releases have actually been built by GitHub Actions - not a user called github-actions. See https://github.com/ungoogled-software/ungoogled-chromium-windows/actions

I'll see if there's a way to make that more clear on the binaries site

**Reply by tomasz1986** [reply]
> It raises a flag for me because I can't '[view all releases from user](https://github.com/github-actions/ungoogled-chromium-binaries/releases)' and gauge the credibility of the source.

In simple language, this means that the binaries have been built on a GitHub server following a user-provided script. This makes them safer than any binaries that are built on some user's computer and only then uploaded to GitHub. In other words, you can be sure that the binaries built by GitHub Actions actually match the source code, which isn't necessarily the case when using user-built binaries.

**Comment by networkException** [comment]
If you take a look now I've updated the display name and link to hopefully make the binaries origin more clear

**Reply by onchov** [reply]
> If you take a look now I've updated the display name and link to hopefully make the binaries origin more clear

Thanks a mill, I did suppose it was some kind of automation but couldn't trace it back for myself with that previous link leading to 404.

**Comment by onchov** [comment]
Clear now, thank you both!

**Comment by Sly-McNasty** [comment]
Hello....I love the idea of browsing without Google looking over my shoulder (facebook as well).   My problem is, I'm not a techy.  I followed a link to download ungoogled-chromium, which brought me to Gethub and now I'm lost.  I'm used to clicking on download, putting the .exe into a folder, scanning it, and then clicking on the .exe to install.  Now I'm dealing with bianaries, codes, and things I don't understand.  In a non-techy verbiage, what do I do after I unzip the .zip file I downloaded.

I'm not completely  computer illiterate.  Way back when, I won the Deans award for making a database for Devry For ALL information on a student from first contact (including grades, parent info, etc.) to the first day in class for every high school in California.  And when I checked 15 years ago they were still using it. (I made it so high schools could be added or deleted).

Can someone PLEASE help me in installing _ungoogled-chromium_ in layman's verbiage.  You can email me at My_Phoney_Email@yahoo.com.

Thank you,
????

**Reply by tomasz1986** [reply]
It seems that you downloaded a zip file from Releases in this repository. This is just the source code, with no possibility to install it, unless you compile the program yourself. If you're looking for Windows installers specifically, please check https://github.com/ungoogled-software/ungoogled-chromium-windows/releases, where you can find .exe installers that you can just double click and install on your system.
